web/app.js

```diff
@@ -2098,9 +2098,13 @@ const PDFViewerApplication = {
   unbindWindowEvents() {
     this._windowAbortController?.abort();
     this._windowAbortController = null;
-    if (AppOptions.get("isInAutomation")) {
+    if (
+      (typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING")) ||
+      AppOptions.get("isInAutomation")
```

Comment:
Should we perhaps change the value of the AppOptions-value when tests are running instead?
https://github.com/mozilla/pdf.js/blob/06800cd966111f7262d3334dfb68c3765bc92740/web/app_options.js#L63-L67

Reply:
But then it'll change the event bus:
https://github.com/mozilla/pdf.js/blob/06800cd966111f7262d3334dfb68c3765bc92740/web/app.js#L384-L387